src/main/java/com/example/a1project/Task.css

```diff
@@ -0,0 +1,72 @@
+.root {
+	-fx-background-color: #ffffff;
+	-fx-background-radius: 25;
+	-fx-border-color: #77B3D4;
+	-fx-border-width: 6 6 6 6;
+	-fx-border-radius: 20;
+}
+
+#addTaskButton { 
+	-fx-background-color: #77B3D4;
+	-fx-background-radius: 20;
+	-fx-font-family: "Avenir Next";
+	-fx-font-size: 48;
+	-fx-font-weight: bold; 
+}
+
+#addTaskButton:hover { 
+	-fx-background-color: #a8daf5;
+}
+
+.label {
+	-fx-font-family: "Avenir Next";
+ 	
+}
+
+.toggle-button {
+	-fx-background-color: #ffffff;
+	-fx-background-radius: 20;
+	-fx-border-color: #77B3D4;
+	-fx-border-width: 3 3 3 3;
+	-fx-border-radius: 20;
+	-fx-font-family: "Avenir Next";
```

Comment:
The same as above could be said for fonts. I think in terms of maintainability the code would benefit from using some form of constant.